packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/Examples.2/ScalableUnits.3.doc.mdx

```diff
@@ -0,0 +1,23 @@
+import { Notice } from '@cerner/terra-docs';
+import ScalableUnits from './ScalableUnits?dev-site-examples';
+
+# Scalable Units
+
+<Notice variant="important" ariaLevel="3">
+
+The supported units for width properties are `px`, `em`, and `rem`. 
+Ensure that the width units are consistent across all properties in the same component. The width properties are
+[columnMinimumWidth](/components/cerner-terra-framework-docs/compact-interactive-list/about#compact-interactive-list-props-table), 
+[columnMaximumWidth](/components/cerner-terra-framework-docs/compact-interactive-list/about#compact-interactive-list-props-table), 
+[width](/components/cerner-terra-framework-docs/compact-interactive-list/about#column), 
+[minimumWidth](/components/cerner-terra-framework-docs/compact-interactive-list/about#column), and 
+[maximumWidth](/components/cerner-terra-framework-docs/compact-interactive-list/about#column).
+Any width units inconsistent with the rest of the component properties are disregarded.
```

Comment:
Does something weird happen if there's no most common one?

Reply:
> Does something weird happen if there's no most common one?

No, it'll work. But if first cells have width in `px`, then second's cell width is in `em`, the width in `em` will be disregarded and a console warning will show up. That will make the second column a column without width. Thise columns just grow and shrink instead of keeping their width.